src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2124,7 +2124,8 @@ console.log(foo + bar) // [!code focus]
 <Code
   code={code}
   lang="js"
-  transformers={[transformerNotationFocus()]} />
+  transformers={[transformerNotationFocus(),[transformerMetaHighlight()]}
```

Comment:
```suggestion
  transformers={[transformerMetaHighlight()]}
```
will commit only the one that makes sense, but not opposed to showing an array if there's a second one that works nicely!